qdrant-landing/content/blog/beta-database-migration-tool.md

```diff
@@ -0,0 +1,38 @@
+---
+draft: false
+title: "Database Migration Tool Beta: One-Command Cluster-to-Cluster Transfers"
```

Comment:
Let's change to "Vector Data Migration Tool". Too long at the moment. No need to add "beta".

Reply:
Why is it not adjusted to  "Vector Data Migration Tool"? @maddie-qdrant